tests/rhtap-demo/rhtap-demo.go

```diff
@@ -387,6 +396,12 @@ var _ = framework.RhtapDemoSuiteDescribe("RHTAP Demo", Label("rhtap-demo"), func
 				Expect(f.AsKubeAdmin.IntegrationController.HaveTestsSucceeded(snapshot)).To(BeTrue(), fmt.Sprintf("tests have not succeeded for snapshot %s/%s", snapshot.GetNamespace(), snapshot.GetName()))
 			})
 
+			It("should validate the pipelineRun is labelled with the name of the Snapshot", func() {
```

Comment:
Shouldn't this block be right after the `It` block that asserts the creation of a snapshot? Because we attach the 'snapshot name' as an annotation to build PLR right after we create the snapshot, so it would look more natural.

Reply:
You are right. Move the block there.